auth0_flutter/lib/src/web/js_interop.dart

```diff
@@ -1,2 +1,79 @@
 // Here go the annotations –from the js package– that allow calling JS code
-// See https://pub.dev/packages/js#usage
\ No newline at end of file
+// See https://pub.dev/packages/js#usage
+
+// ignore_for_file: non_constant_identifier_names
+
+@JS('auth0')
+library auth0;
+
+import 'dart:js_util';
+
+import 'package:js/js.dart';
+
+@JS()
+@staticInterop
+@anonymous
+abstract class AuthorizationParams {
+  external factory AuthorizationParams(
+      {final String? audience, final String? redirect_uri});
+}
+
+extension on AuthorizationParams {
+  external String? audience;
+  external String? redirect_uri;
+
+  // https://github.com/dart-lang/sdk/issues/38445
+  // Workaround of setting property names doesn't seem to work. Tried:
+  // @JS('redirect_uri')
+  // external String? get redirectUri;
+
+  // @JS('redirect_uri')
+  // external set redirectUri(final String? redirectUri);
+}
+
+/// Rebuilds the input object, omitting values that are null
+@JS()
+T stripNulls<T extends Object>(final T obj) {
```

Comment:
Is the `@JS()` annotation necessary here?

Reply:
Doesn't look like it! Will remove 👍🏻

Reply:
Removed it. I also moved it out into the plugin file as a private method, keeping the interop file just for interop stuff.